### PR TITLE
Fix yarn start warnings

### PR DESCRIPTION
**Pull request by lucasponce**
Fixes https://github.com/kiali/kiali/issues/3267

```
Note that the development build is not optimized.
To create a production build, use yarn build.
Compiling...
Files successfully emitted, waiting for typecheck results...
Compiled successfully!

You can now view @kiali/kiali-ui in the browser.

  Local:            http://localhost:3000
  On Your Network:  http://192.168.1.145:3000

Note that the development build is not optimized.
To create a production build, use yarn build.
```

### CODE REVIEW

**Review by mtho11 — COMMENTED** [review]
Thanks for taking care of that @lucasponce, it was annoying at least and bothersome when trying to read the logs before the warnings.

**Review by jotak — APPROVED** [review]
lgtm , I haven't tested, can do it tomorrow

File: src/components/Filters/StatefulFilters.tsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ import {
   ToolbarGroup,
   ToolbarItem,
   ToolbarSection,
-  SelectOptionObject
+  SelectOptionObject,
+  Button
```
Comment (israel-hdez) [inline src/components/Filters/StatefulFilters.tsx:18]: Minor: Breaks alphabetization.

**Review by israel-hdez — APPROVED** [review]
UI looks good and I no longer see the warnings when running `yarn start`.
LGTM.

There is a small comment, but not a blocker.

**Review by jotak — APPROVED** [review]
Tests LGTM

File: src/components/Filters/StatefulFilters.tsx (hunk 1 above)
Comment (lucasponce) [reply src/components/Filters/StatefulFilters.tsx:18]: I think in the UI alphabetization is broken everywhere so perhaps I'll defer that change.

**Pull request merged**